Client/game_sa/CGameSA.cpp

```diff
@@ -34,16 +34,46 @@ unsigned long* CGameSA::VAR_Framelimiter;
 
 unsigned int OBJECTDYNAMICINFO_MAX = *(uint32_t*)0x59FB4C != 0x90909090 ? *(uint32_t*)0x59FB4C : 160;            // default: 160
 
+static int m_iVehicleSunGlare;
+#define HOOK_CAutomobile_OnVehiclePreRender 0x6ABCFD
```

Comment:
I think it's better to put this in CMultiplayerSA.cpp. We use hooks there. Im not sure

Reply:
An Example? i don't think i can get the `HOOK_CAutomobile_OnVehiclePreRender`,etc directly from CMultiplayerSA.cpp to `CGameSA.cpp`

Reply:
I see, some hooks are scoped there. Maybe, it's better to move this hooks in CVehicleSA::StaticSetHooks

Reply:
@TheNormalnij Should be cool now, i moved the hooks and corespond code to the CVehicleSA, check it